.ci/configure_crio_for_kata.sh

```diff
@@ -16,14 +16,19 @@ minor_crio_version=$(crio --version | head -1 | cut -d '.' -f2)
 if [ "$minor_crio_version" -ge "12" ]; then
 	echo "Configure runtimes map for RuntimeClass feature"
 	echo "- Set runc as default runtime"
-	sudo sed -i 's!runtime_path =.*!runtime_path = "/usr/local/bin/crio-runc"!' "$crio_config_file"
-	sudo sed -i 's!runtime_type =.*!runtime_type = "oci"!' "$crio_config_file"
+	if [ -n $(grep -q "crio.runc" "$crio_config_file") ]; then
+		sudo sed -i 's!runtime_path =.*!runtime_path = "/usr/local/bin/crio-runc"!' "$crio_config_file"
```

Comment:
I think this single line should go outside the `if` as it is only replacing the default cri-o path with the one that we use, so even if this runs several times, we shouldn't get a duplicated entry.

Reply:
But,,, IMO, once you configured once, adding `kata-runtime` as another option. There will be two runtime_path, like the following:
```
[crio.runtime.runtimes.runc]
runtime_path = "/usr/local/bin/crio-runc"
runtime_root = "/run/runc"
runtime_type = "oci"
[crio.runtime.runtimes.kata]
runtime_path = "/usr/local/bin/kata-runtime"
runtime_root = "/run/vc"
runtime_type= "oci"
```
And this single line is instructed to replace any line starting with `runtime_path =`. 
So, it will also replace what's configured for `kata-runtime`, not just `runc`.

Reply:
I think @Pennyzct is right - the `grep` is specific to runc, but the `sed` is not, so would also change the kata path if it were already in the file (like it might be on a bare metal slave, which is what this PR is trying to fix).